app/uk/gov/hmrc/pillar2submissionapi/connectors/GIRConnector.scala

```diff
@@ -18,6 +18,7 @@ package uk.gov.hmrc.pillar2submissionapi.connectors
 
 import play.api.Logging
 import play.api.libs.json.Json
+import play.api.libs.ws.JsonBodyWritables.writeableOf_JsValue
```

Comment:
This is OK, just want to highlight the new syntax. Sometimes it's a little nicer if the instance names are a bit unpleasant. 
```suggestion
import play.api.libs.ws.JsonBodyWritables.given
```